requirements/pypi-optional-test.txt

```diff
@@ -4,3 +4,4 @@ pytest>=6.0
 pytest-cov
 pytest-mock
 pytest-pyvista
+pandas
```

Comment:
@tkoyama010 Would you mind removing `pandas` from this file, then creating a new file called `requirements/pypi-optional-exam.txt` which contains:
```bash
fastparquet
pandas
```
Then add the following entry to the `pyproject.toml`:
![image](https://github.com/bjlittle/geovista/assets/2051656/11c6ee2c-7b0d-44f4-b588-6d7e75d3ad22)

This will allow users to specify the `example` dependencies independently of the `test` dependencies i.e.,
```bash
$ pip install geovista[exam]
```
or
```bash
$ pip install geovista[exam,cmap]
```
etc

Reply:
Good point. Thanks!
```suggestion
```

Reply:
@tkoyama010 Sorry ... could you add the entry mentioned above to the `pyproject.toml`, thanks :beers: